client/src/components/JurisdictionAdmin/generateSheets.ts

```diff
@@ -795,3 +797,128 @@ export const downloadTallyEntryLoginLinkPrintout = async (
   )
   return doc.output() // returned for test snapshots
 }
+
+const AUDIT_BOARD_SIGN_MAX_CONTEST_NAME_LENGTH = 40
+const AUDIT_BOARD_SIGN_MAX_SURNAME_LENGTH = 20
+const AUDIT_BOARD_SIGN_TITLE_LENGTH_XL_FONT_CUTOFF = 12
+const AUDIT_BOARD_SIGN_TEXT_COLOR = Colors.GRAY3
+const AUDIT_BOARD_SIGN_TITLE_COLORS = [
+  Colors.GREEN1,
+  Colors.VIOLET1,
+  Colors.ORANGE1,
+  Colors.VERMILION1,
+  Colors.BLUE1,
+  Colors.COBALT1,
+  Colors.GOLD1,
+  Colors.SEPIA1,
+  Colors.DARK_GRAY1,
+  Colors.INDIGO1,
+]
+const DEFAULT_AUDIT_BOARD_SIGNS = [
+  { title: 'No Vote', subtitle: '(undervote, overvote, blank, etc.)' },
+  { title: 'For Review', subtitle: '(undetermined, duplicated, etc.)' },
+]
```

Comment:
I think we can remove parens from subtitles since they are already styled in a way that makes it clear they are less important. Also we might want to capitalize the first letter of the subtitle if we remove the parens